Libraries/QtVgCommon/vgTimeMap.h

```diff
@@ -16,6 +16,9 @@
 template <typename Value>
 class vgTimeMap : public QMap<vgTimeStamp, Value>
 {
+  template <typename Iterator, typename MapPtr>
+  static Iterator find(MapPtr map, vgTimeStamp pos, vg::SeekMode direction);
```

Comment:
Not that it matters much, but when I did the Qt5 port independently in my own code, I put this helper function in a `detail` namespace rather than inside the class to avoid having an unnecessary double-template.

Reply:
Eh... the double template doesn't bother me overly :slightly_smiling_face:. I like having it in the class because that makes it actually-private.